config/locales/en.yml

```diff
@@ -810,55 +825,33 @@ funding:
-          - Primarily help people in the UK
-          readMore: >
-            If you’re not sure about any of these please go and check with colleagues before proceeding. For more information, please <a href="%s">see the full detailed eligibility criteria for this programme</a>
-    strand2:
-      title: We are already highly digital, and want to scale our impact
-      shortTitle: Scale your impact
-      teaserDescription: Grants of up to £500,000 for tech for good organisations that have already launched promising services
-      introduction: Digital Fund Strand 2 offers grants of up to £500,000 for newer organisations that have already launched promising services that use digital to achieve scale or impact.
-      startApplying: Start applying for strand 2
-      eligibilityDetail:
-        title: Request a call
-        intro: >
-          <p><strong>Your time is precious — we don’t want to waste it</strong>.</p>
-          <p>Therefore, we’ll need to ask a couple of quick questions about your organisation and your funding needs in advance of our call. We’ll also need your contact details so we can arrange a convenient time for us to chat. This shouldn’t take more than a few minutes.</p>
-          <p>To get started, please answer the first question below.</p>
-          <p><strong>Do you currently operate a service that benefits at least 50 people every week?</strong></p>
-        unsure:
-          title: I'm not sure
-          intro: 'This programme is specifically designed to help organisations that:'
-          criteria:
-          - Are past the idea or prototype stage, and that are being used to help at least 50 people a week already
-          - Use digital to open the possibility to large scale impact. This doesn’t necessarily mean that they are first and foremost and app, though, they might be a project that uses technology to arrange volunteers in many different places
-          - Have some strong digital skills within the founding team
-          readMore: >
-            For more information, please see the <a href="%s">full detailed eligibility criteria</a> for this programme.
+    eligibilityChecker:
+      title: Request a call
+      subtitle: Your time is precious — we don’t want to waste it
+      introduction: Please assist us by giving us just a bit of information about your idea and your organisation. We need to know how to get in touch with you, and we need to know that arranging a call is appropriate. To get started, please answer the first question below.
+      unsureTitle: I'm not sure
+      strand1:
+        question: Are you from a UK-based, registered charity which had income of over £500,000 last year?
+        unsure: >
+          <p>This programme is specifically designed to help organisations that:</p>
+          <ul>
+            <li>Are medium or large in size</li>
+            <li>Have structures and ways of working were developed before the internet became widespread</li>
```

Comment:
"Have structures and ways of working were" should be "Have structures and ways of working which were"